backend/src/controllers/UserController.js

```diff
@@ -0,0 +1,42 @@
+const connection = require('../database/connection');
+
+module.exports = {
+    async index(request, response) {
+        const user = await connection('user').select('*');
+    
+        return response.json(user);
+    },
+
+    async create(request, response){
```

Comment:
eslint: [space-before-function-paren](http://eslint.org/docs/rules/space-before-function-paren) (and after too)